modules/virt-restoring-vm-from-snapshot-cli.adoc

```diff
@@ -12,6 +12,12 @@ You can restore an existing virtual machine (VM) to a previous configuration by
 
 * Power down the VM you want to restore.
 
+* Optional: Adjust what happens if the target VM is not ready. To do so, set the `targetReadinessPolicy` parameter in the `vmrestore` YAML configuration to one of the following values:
```

Comment:
maybe this should be as a subpoint of the previous one of powerdown the vm.
And I guess maybe need to explain in this case that vm ready means vm stopped.